doc/plot_conf.yml

```diff
@@ -0,0 +1,117 @@
+# 
+# Documentation's example plot settings
+#
+# A "default" key allows setting default values for each entry. All the
+# settings defined there can be overriden for specific methods.
+#
+# Each entry needs the following keys:
+#   trace: a trace *object* (use !call or !var tags provided by lisa.utils.Serializable)
+#   kwargs [optional]: keyword arguments to pass to the plot function. This may
+#       be mandatory for some plot functions that need a task or cpu name for
+#       example. Bear in mind that YAML is "typed", so 0 is not the same as "0"
+#   hide [optional]: Do not produce an example plot, in case no trace with
+#   necessary events is available for example.
+
+
+doc-plot-conf:
+    plots:
+        # Default values for the other entries
+        default:
+            trace: &trace0 !call:lisa.trace.Trace
+                trace_path: !env:interpolate $LISA_HOME/doc/traces/trace.dat
+                plat_info: !call:lisa.platforms.platinfo.PlatformInfo.from_yaml_map
+                    path: !env:interpolate $LISA_HOME/doc/traces/plat_info.yml
+
+        TasksAnalysis.plot_task_residency:
+            kwargs:
+                task: &task big_0
```

Comment:
Not a YAML guru - is there a clean way to declare those handles outside of a plot entry? IMO it would read better (maybe something like a "plot_parameters" entry). If that's too convoluted then that's fine, I'm not too attached to the idea.

Reply:
It can be defined outside, but I'd rather not pollute the file with extra keys that don't have direct influence (merely adding something in "plot_parameters" will not trigger anything).
Maybe we could establish a convention across LISA conf files and "reserve" a top-level `scratch` key if we want to do this kind of things.